cmake/Reprojucer.cmake

```diff
@@ -530,6 +531,11 @@ function(jucer_export_target exporter)
           )
         endif()
 
+      elseif(tag STREQUAL "PKGCONFIG_LIBRARIES")
+        string(REPLACE " " ";" value "${value}")
+        list(APPEND JUCER_PROJECT_LINUX_PACKAGES ${value})
+        set(JUCER_PROJECT_LINUX_PACKAGES ${JUCER_PROJECT_LINUX_PACKAGES} PARENT_SCOPE)
```

Comment:
@MartyLake: we can simply add these extra "pgk-config libraries" to `JUCER_PROJECT_LINUX_PACKAGES` because the infrastructure is already setup, see https://github.com/McMartin/JUCE.cmake/blob/1d4f04a14aa7b86de93643b76b1294c0395f3b46/cmake/Reprojucer.cmake#L1406.